src/core_init_atmosphere/mpas_init_atm_static.F

```diff
@@ -750,10 +759,13 @@ subroutine init_atm_static(mesh, dims, configs)
 
             all_pixels_mapped_to_halo_cells = .true.
 
-            do j = tile_bdr + 1, tile_ny + tile_bdr, 1
-                do i = tile_bdr + 1, tile_nx + tile_bdr, 1
+            do j = supersample_fac_30s * tile_bdr + 1, supersample_fac_30s * (tile_ny + tile_bdr), 1
+                do i = supersample_fac_30s * tile_bdr + 1, supersample_fac_30s * (tile_nx + tile_bdr), 1
+
+                    ii = (i - 1) / supersample_fac_30s + 1
```

Comment:
Would it be reasonable to change these variables from "ii" and "jj" to something more descriptive? "tile_x" and "tile_y" or something like that? I have a vendetta against ii and jj as indexing variables, but maybe it's more trouble to change here than it's worth.

Reply:
In my opinion, the index variable names are probably fine as they are. The code you've referenced mirrors what is done in a more general routine, [init_atm_map_static_data](https://github.com/MPAS-Dev/MPAS-Model/blob/v8.0.1/src/core_init_atmosphere/mpas_init_atm_static.F#L1200), and so I think there's value in keeping the variable names consistent. Not that I have anything against descriptive variable names, but do we then insist that the loop variables `i` and `j` be given better names? `ii` and `jj` are the counterparts of `i` and `j`, but in non-supersampled indexing space.